backend/app/core/review_rag.py

```
@@ -52,49 +52,56 @@
+                        "Authorization": f"Bearer {self.api_key}",
+                        "Content-Type": "application/json",
+                    },
                     json={"model": self.model, "input": batch},
                     timeout=120.0,
                 )
                 resp.raise_for_status()
-                batch_embeddings = resp.json()["embeddings"]
-                embeddings.extend(batch_embeddings)
+                data = resp.json()["data"]
+                embeddings.extend([d["embedding"] for d in data])
             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)